test/modules.js

```diff
@@ -46,7 +46,7 @@ import '@loaders.gl/tiles/test/index.js';
 
 // Geospatial Formats
 import '@loaders.gl/flatgeobuf/test/index.js';
-import '@loaders.gl/geopackage/test/index.js';
+import '@loaders.gl/geopackage/test/index.ts';
```

Comment:
I forget why we need to type out index / extensions here? Perhaps that was part of some experiment and is not needed.

Reply:
I don't know... but it doesn't seem to work with .ts anyways